temporian/test/prototype_test_numpy.py

```diff
@@ -151,41 +172,6 @@ def setUp(self) -> None:
             sampling=sampling,
         )
 
-        # expected output event. Select "costs" feature only
-        # self.expected_output_event = NumpyEvent(
-        #     data={
-        #         ("A", 1): [
-        #             NumpyFeature(
-        #                 name="costs",
-        #                 sampling=sampling,
-        #                 data=np.array([11, 12, 13]),
-        #             ),
-        #         ],
-        #         ("A", 2): [
-        #             NumpyFeature(
-        #                 name="costs",
-        #                 sampling=sampling,
-        #                 data=np.array([9]),
-        #             ),
-        #         ],
-        #         ("B", 2): [
-        #             NumpyFeature(
-        #                 name="costs",
-        #                 sampling=sampling,
-        #                 data=np.array([5, 6]),
-        #             ),
-        #         ],
-        #         ("B", 3): [
-        #             NumpyFeature(
-        #                 name="costs",
-        #                 sampling=sampling,
-        #                 data=np.array([1, 2]),
-        #             ),
-        #         ],
-        #     },
-        #     sampling=sampling,
-        # )
-
```

Comment:
Remove

Reply:
it's in read, it means it already has been removed. Almost got me there

Reply:
lol. oops.